xdsl/printer.py

```diff
@@ -7,6 +7,7 @@
 from itertools import chain
 from typing import Any, TypeVar, cast
 
+import numpy as np
```

Comment:
Were we already depending on numpy? This seems like a pretty sizable import if not

Reply:
I think only JAX uses it so far.